### DISCUSSION TITLE
Does it support Android 4.15 or less?

### DISCUSSION THREAD

**Discussion by Rgoogle** [gojue/ecapture#226] [General]
Does it support Android 4.15 or less?
 I'm Xiaomi 8 ，it kernel version is 4.9. when i  run it.I don't know why it said 4.9 is bigger than 4.15.

**Comment by cfc4n** [comment]
> Does it support Android 4.15 or less? I'm Xiaomi 8 ，it kernel version is 4.9. when i run it.I don't know why it said 4.9 is bigger than 4.15.

eCapture can not support Linux/Android Kernel 4.15, and also support  Kernel 5.5 or newer due `bpf_probe_read` function return error(-14) on arm64 kernel.

see https://yhbt.net/lore/all/YHGMFzQlHomDtZYG@kroah.com/t/ for more detail.

via : https://github.com/iovisor/bcc/issues/3175
